libs/core/src/lib/button/base-button.ts

```diff
@@ -61,6 +61,10 @@ export class BaseButton {
     @Input()
     fdMenu = false;
 
+    /** Sets aria-label attribute for button */
+    @Input()
+    ariaLabel: string;
```

Comment:
i don't think this is needed. See above comment.

Reply:
comments here https://github.com/SAP/fundamental-ngx/pull/6297#discussion_r690258903